src/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Box.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace System.Threading.Tasks
+{
+    /// <summary>Utility class for allocating value types as heap variables.</summary>
+    internal class Box<T>
```

Comment:
Is ```StrongBox<T>``` available in the contracts referenced by this library?  If so, that could be used instead (could be done as a separate cleanup after committing this).